ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/MLChatAgentRunner.java

```diff
@@ -145,7 +145,7 @@ public void run(MLAgent mlAgent, Map<String, String> params, ActionListener<Obje
 
                 runAgent(mlAgent, params, listener, memory, memory.getConversationId());
             }, e -> {
-                log.error("Failed to get chat history", e);
+                log.error("Failed to run agent " + mlAgent.getName(), e);
```

Comment:
Proper use of log facility needs to use placeholders pattern instead of concatenation or manual interpolation: `log.error("Failed to run agent: {} ",  mlAgent.getName(), e)`